### PR TITLE
chore: update all secrethub with keeper command

### PR DESCRIPTION
**Pull request by sursak123**
chore: update all secrethub with keeper command

secrethub orb will not be used any more and has to be replaced with keeper

**Description**

Updated keeper orb and keeper urls in the jobs

**Additional context**

---
📚&nbsp;&nbsp;View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-lrdlfwsjrt.chromatic.com)

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@3.3.1-cicd-secrethub-to-keeper-f2532c3
# or
yarn add @gravitee/ui-policy-studio-angular@3.3.1-cicd-secrethub-to-keeper-f2532c3
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@3.3.1-cicd-secrethub-to-keeper-f2532c3
# or
yarn add @gravitee/ui-particles-angular@3.3.1-cicd-secrethub-to-keeper-f2532c3
```

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 version: 2.1
-
 orbs:
-  secrethub: secrethub/cli@1.1.0
+  keeper: gravitee-io/keeper@0.6.1
   gh: circleci/github-cli@2.0.0
-
+
```
Comment (gaetanmaisse) [inline .circleci/config.yml:5]: We need to remove the extra space

Comment (gaetanmaisse) [reply .circleci/config.yml:5]: @sursak123 can you please update your IDE to not add these extra spaces and remove them from your commit?

**Review by gaetanmaisse — CHANGES_REQUESTED** [review]

**Review by gaetanmaisse — APPROVED** [review]

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-particles-angular@3.3.1-cicd-secrethub-to-keeper-1a50e8a
# or
yarn add @gravitee/ui-particles-angular@3.3.1-cicd-secrethub-to-keeper-1a50e8a
```

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**
A prerelease version of this package has been published! 🚀
To use it in your project run the following lines:
```shell
npm install @gravitee/ui-policy-studio-angular@3.3.1-cicd-secrethub-to-keeper-1a50e8a
# or
yarn add @gravitee/ui-policy-studio-angular@3.3.1-cicd-secrethub-to-keeper-1a50e8a
```

**Pull request merged**